CMakeLists.txt

```diff
@@ -19,6 +19,10 @@ cmake_minimum_required(VERSION 3.10)
 
 project(Video_Processing_Framework)
 
+if (DEFINED ENV{PATH_TO_SDK})
```

Comment:
This change doesn't relate to PR topic, please put it to separate PR or remove from current PR.